common/source/docs/common-smart-battery-landingpage.rst

```diff
@@ -0,0 +1,45 @@
+.. _common-smartbattery-landingpage:
+
+=============================================
+Smart Batteries
+=============================================
+
+ArduPilot supports three types of "smart batteries" that communicate using `SMBus <https://en.wikipedia.org/wiki/System_Management_Bus>`__: the Maxell battery, the `Solo battery <https://www.amazon.com/3DR-BT11A-Solo-Smart-Battery/dp/B00X97EXEU>`__, and the `Rotoye BatMon <https://rotoye.com/>`__.
```

Comment:
should make a bulleted list that can be expanded easily instead of a long sentence